ApplicationView/ApplicationController.cs

```diff
@@ -1724,13 +1724,15 @@ public static SystemWindow LoadRootWindow(int width, int height)
 
 			systemWindow.AddChild(overlay);
 
-			var imageWidget = new ImageWidget(CreateBackgroundImage((int)systemWindow.Width, (int)systemWindow.Height));
-			overlay.AddChild(imageWidget);
+			var imagePath = AggContext.StaticData.MapPath(Path.Combine("Images", "splash.png"));
+			var sourceImage = AggContext.StaticData.LoadImage(imagePath);
```

Comment:
Just for reference...

StaticData.LoadImage calls takes StaticData relative paths. The extra call to MapPath is unnecessary

Reply:
Thank you. Fixed